samples/solidity/contracts/TokenFactory.sol

```diff
@@ -38,14 +38,15 @@ contract TokenFactory is Context {
         string memory name,
         string memory symbol,
         bool is_fungible,
-        bytes calldata data
+        bytes calldata data,
+        string memory uri
```

Comment:
I think the connector is going to need to be able to figure out if URI is supported. Potentially that means adding ERC165 and "supportsInterface" to this contract similar to how it's present in the ERC20/ERC721 contracts.